src/databaseaccessor/commentdb.cpp

```diff
@@ -0,0 +1,113 @@
+#include "commentdb.h"
+#include <QDebug>
+CommentDb::CommentDb(): Accessor()
+{
+}
+
+void CommentDb::addCommentsToDb(const QVector<Comment> &comments)
+{
+    for (auto &i : comments)
+    {
+        execQuery(addCommentQuery(i));
+    }
+    query.finish();
+}
+
+void CommentDb::deleteCommentFromDb(const int commentLine, const QString commentFile)
+{
+    execQuery(deleteCommentQuery(commentLine, commentFile));
+    query.finish();
+}
+
+void CommentDb::deleteCommentsFromDb(const QString commentFile)
+{
+    execQuery(deleteAllCommentsInFileQuery(commentFile));
+}
+
+QVector<Comment> CommentDb::getAllCommentsFromFile(const QString filename)
+{
+      execQuery(numberOfCommentInFileQuery(filename));
+      query.first();
+      int count_of_messages =query.value(0).toInt();
```

Comment:
In accordance to code convention 
int countOfMessages

Reply:
Refactor **count_of_messages** plus add space after equals sign (=)